docs/examples/Transformer Examples.ipynb

```diff
@@ -32,7 +32,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 166,
+   "execution_count": 330,
```

Comment:
please clear output, restart kernel and rerun from scratch to reset execution count